front/lib/models/assistant/agent.ts

```diff
@@ -166,3 +170,8 @@ Workspace.hasMany(AgentConfiguration, {
 AgentGenerationConfiguration.hasOne(AgentConfiguration, {
   foreignKey: { name: "generationConfigurationId", allowNull: true }, // null = no generation set for this Agent
 });
+
+AgentConfiguration.belongsTo(AgentGenerationConfiguration, {
```

Comment:
This one is exactly equivalent to the one above and should be removed. You can add the `as` in the `hasOne` declaration :+1:

Reply:
@spolu If I remove it I get an error 
`EagerLoadingError [SequelizeEagerLoadingError]: agent_generation_configuration is not associated to agent_configuration!`

Reply:
Looking at the code it looks like it's needed : 

<img width="985" alt="Capture d’écran 2023-09-11 à 10 29 22" src="https://github.com/dust-tt/dust/assets/3803406/004f6056-f2f5-4090-bbe9-a8c1bffc65db">